azure-app-service/massdriver.yaml

```diff
@@ -144,29 +143,13 @@ params:
           description: Azure App Service Plan size
           type: string
           oneOf:
-            - title: 1 vCPU, 1.75GB RAM, scales up to 3 instances
-              const: B1
-            - title: 2 vCPU, 3.5GB RAM, scales up to 3 instances
-              const: B2
-            - title: 4 vCPU, 7GB RAM, scales up to 3 instances
-              const: B3
-            - title: 1 vCPU, 1.75GB RAM, scales up to 10 instances
+            - title: 1 vCPU, 1.75GB RAM
```

Comment:
@mclacore I think we can do a follow-up PR for these. Either a list or we can do a mini hackathon together and get it into Satellite. 
🤙